bin/pre-commit-hook.js

```diff
@@ -17,7 +18,7 @@ const files = execSync( 'git diff --cached --name-only' )
 	.map( ( name ) => name.trim() )
 	.filter( ( name ) => name.endsWith( '.js' ) || name.endsWith( '.jsx' ) );
 
-const lintResult = spawnSync( 'eslint-eslines', [ ...files, '--', '--diff=index' ], {
```

Comment:
Well, I feel pretty stupid now 😅 
I think you're right, I was in `master` at that time. I've reverted this change and tried again, and it works.